pkg/metaserver/agent/pod/runtime.go

```diff
@@ -55,23 +55,27 @@ type labeledContainerInfo struct {
 	PodUID        kubetypes.UID
 }
 
-type RuntimePodFetcher struct {
+type RuntimePodFetcher interface {
+	GetPods(all bool) ([]*RuntimePod, error)
+}
+
+type RuntimePodFetcherImpl struct {
 	runtimeService cri.RuntimeService
 }
 
-func NewRuntimePodFetcher(conf *config.Configuration) (*RuntimePodFetcher, error) {
+func NewRuntimePodFetcher(conf *config.Configuration) (*RuntimePodFetcherImpl, error) {
```

Comment:
construct function return interface `RuntimePodFetcher` directly to avoid mistake use

Reply:
fixed